plugins/GSdx/GSdx.rc

```diff
@@ -236,6 +236,8 @@ BEGIN
     COMBOBOX        IDC_ADAPTER,71,55,166,118,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
     LTEXT           "Renderer:",IDC_STATIC,6,72,34,8
     COMBOBOX        IDC_RENDERER,71,70,166,118,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+    COMBOBOX        IDC_MIPMAP,71,100,166,118,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+    LTEXT           "Mipmap (Insert):",IDC_MIPMAP_TEXT,6,102,53,8
```

Comment:
Thanks, I have also made the string changes suggested by @FlatOutPS2 on Linux GUI.